File: backend/Multibanca.Domain.Models/Multibanca/BBVA/devolucion_vb_comercial.cs
using Common.Domain.Models;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Multibanca.Domain.Models.Multibanca.BBVA
{
    public class devolucion_vb_comercial : base_auditoria
    {
        public int id { get; set; }
        public long id_expediente { get; set; }
        public string id_actividad { get; set; } = "ACT_DEVOLUCION_VB_COMERCIAL";
        public bool? cliente_desiste { get; set; }
        public string? motivo_cierre { get; set; }
        public string? observaciones { get; set; }
    }
}

File: frontend/src/features/actividades/devolucion_vb_comercial/models/devolucion_vb_comercial.ts
export interface DevolucionVbComercialData {
  id: number;
  idExpediente: number;
  idActividad: string;
  clienteDesiste: boolean | null;
  motivoCierre?: string | null;
  observaciones?: string | null;
}

export interface DevolucionVbComercialForm {
  actividad: DevolucionVbComercialData;
}
